sw/device/examples/sram_program/BUILD

```diff
@@ -31,6 +32,13 @@ opentitan_ram_binary(
         "//sw/device/lib/base:macros",
         "//sw/device/lib/dif:sram_ctrl",
         "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing:pinmux_testutils",
         "//sw/device/lib/testing/test_framework:check",
     ],
 )
+
+opentitan_sram_fpga_cw310_test(
```

Comment:
the gdb script should appear here, right?

Reply:
Done